.github/workflows/base-lambdas-edge-deploy.yml

```diff
@@ -0,0 +1,99 @@
+name: 'Z-BASE Lambdas: action to deploy a single lambda'
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+      python_version:
+        required: true
+        type: string
+      build_branch:
+        required: true
+        type: string
+        default: main
+      sandbox:
+        required: true
+        type: string
+        default: main
+      lambda_handler_name:
+        required: true
+        type: string
+      lambda_aws_name:
+        required: true
+        type: string
+    secrets:
+      AWS_ASSUME_ROLE:
+        required: true
+
+permissions:
+  pull-requests: write
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+jobs:
+  python_deploy_lambda:
+    name: Deploy lambda ${{ inputs.lambda_handler_name }} to ${{ inputs.sandbox }}
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+
+    steps:
+      - name: Display passed variables
```

Comment:
Can we reuse `.github/workflows/base-lambdas-reusable-deploy.yml` here?

Reply:
We could modify the lambda layer script to return if an empty string is passed, bypassing the layer application. That way we could reuse the workflow noga mentioned above